pkg/install/aws.go

```diff
@@ -6,8 +6,8 @@ import (
 	"os"
 
 	"github.com/argoproj/argo-cd/v2/util/cli"
-	"sigs.k8s.io/cluster-api-provider-aws/cmd/clusterawsadm/cmd"
-	credentials2 "sigs.k8s.io/cluster-api-provider-aws/cmd/clusterawsadm/credentials"
+	"sigs.k8s.io/cluster-api-provider-aws/v2/cmd/clusterawsadm/cmd"
```

Comment:
we'll need to validate install command as well

Reply:
Works... I tested it

Reply:
Thanks. In fact, I think this could be backported to v0.10 too - where we are not using CAPA v2 (will require testing).

Reply:
Arlon init installs latest version of capa, so capa v2 on that release is already tested. CC: @ShaunakJoshi1407